server/qrops.go

```diff
@@ -0,0 +1,273 @@
+package server
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"image"
+	"net/url"
+	"strconv"
+
+	"github.com/yeqown/go-qrcode/v2"
+	"github.com/yeqown/go-qrcode/writer/standard"
+	"go.uber.org/zap"
+
+	"github.com/status-im/status-go/images"
+	"github.com/status-im/status-go/multiaccounts"
+)
+
+type WriterCloserByteBuffer struct {
+	*bytes.Buffer
+}
+
+func (wc WriterCloserByteBuffer) Close() error {
+	return nil
+}
+
+func NewWriterCloserByteBuffer() *WriterCloserByteBuffer {
+	return &WriterCloserByteBuffer{bytes.NewBuffer([]byte{})}
+}
+
+func ToLogoImageFromBytes(imageBytes []byte, padding int) ([]byte, error) {
+	img, _, err := image.Decode(bytes.NewReader(imageBytes))
+	if err != nil {
+		return nil, fmt.Errorf("decoding image failed: %v", err)
+	}
+	paddedImg := images.AddPadding(img, padding)
+	circle := images.CreateCircle(img)
+	centeredImg := images.PlaceCircleInCenter(paddedImg, circle)
+	resultBytes, err := images.EncodePNG(centeredImg)
+	if err != nil {
+		return nil, fmt.Errorf("encoding PNG failed: %v", err)
+	}
+	return resultBytes, nil
+}
+
+func GetLogoImage(multiaccountsDB *multiaccounts.Database, keyUID string, imageName string) ([]byte, error) {
+	var (
+		imageFolderBasePath = "../_assets/tests/"
+		logoFileStaticPath  = imageFolderBasePath + "status.png"
+		padding             int
+		LogoBytes           []byte
+	)
+
+	identityImageObjectFromDB, err := multiaccountsDB.GetIdentityImage(keyUID, imageName)
+
+	if err != nil {
+		return nil, err
+	}
+
+	staticLogoFileBytes, _ := images.ImageToBytes(logoFileStaticPath)
```

Comment:
Should the error here be handled or logged?

Reply:
hmmmm, yes it wouldn't hurt to log it, I'll refactor, thanks for looking at this @richard-ramos